build-scripts/config_common.cmake

```diff
@@ -351,7 +347,7 @@ if (WAMR_BUILD_STRINGREF EQUAL 1)
 endif ()
 if (WAMR_BUILD_PERF_PROFILING EQUAL 1 OR
     WAMR_BUILD_DUMP_CALL_STACK EQUAL 1 OR
-    WAMR_BUILD_GC EQUAL 1 OR WAMR_BUILD_GC_BINARYEN EQUAL 1)
+    WAMR_BUILD_GC EQUAL 1)
   # Enable AOT/JIT stack frame when perf-profiling, dump-call-stack,
   # gc or gc-binaryen is enabled
```

Comment:
Maybe remove the "gc-binaryen" in the comment too?

Reply:
Done, thanks